flask-api/flaskr/candidate.py

```diff
@@ -192,9 +192,6 @@ def update(candidate_id):
     if v_event is None:
         abort(400, 'Bad Request')
 
-    if candidate_order != None and candidate_order > count:
-        abort(400, 'Bad Request: Invalid number') # IDK if there's a better message, but this is here just for debugging purposes if someone spaghets
-

```

Comment:
Turns out this doesn't work as intended. I can't be bothered implementing proper checks. Business logic too hard.

Reply:
what was it meant to do?

Reply:
Say you have a party of 3 candidates and you're trying to modify the candidate order.
The code was supposed to ensure that you can't put a number larger than 3.
But when combined with other functionality, it doesn't work as intended. (e.g. always throwing error 400)